### PR TITLE
PR-5229 Add CI/CD

### PR DESCRIPTION
**Pull request by reweeden**
I tested the 3.6/3.9 artifact in the rain dev account. I'm thinking of using calver actually since we're not really releasing code with an API.

### CODE REVIEW

**Review by gjclark — APPROVED** [review]

**Review by mattp0 — APPROVED** [review]
maybe a different PR or this one. But can we add a readme on making a new release and bit about what it is.

**Comment by reweeden** [comment]
Good call on the readme

**Review by mckadesorensen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
 # Python GDAL lambda layer
+A build pipeline for creating a lambda layer with GDAL and the python bindings
+installed. Ready to use zip packages are available on each release.
```
Comment (gjclark) [inline README.md]:
```suggestion
installed.  Ready-to-use zip packages are available on each release.
```

Wording is a little clearer this way?

**Review by gjclark — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (reweeden) [reply README.md]: Sure

Comment (reweeden) [reply README.md]: I won't add the extra space though :P

**Pull request merged**